ChangeLog.md

```diff
@@ -4,6 +4,9 @@
 
 ## 1.10
 
+ * Add backtrace/ptrace pretty-printers for mtev_hash_table and
+   mtev_http*_request values
```

Comment:
Should wrap the code names with backticks to prevent formatting weirdness due to the unbalanced '*'

Reply:
indeed. _done_